vpn/__init__.py

```diff
@@ -0,0 +1,70 @@
+"""VPN Extension
+
+Connect or disconnect from a network manager VPN profile"""
+
+
+from albert import *
+from collections import namedtuple
+import subprocess
+
+md_iid = "0.5"
+md_version = "0.5"
+md_id = "vpn"
+md_name = "VPN"
+md_description = "Manages VPN (via nmcli)"
+md_license = "MIT"
+md_url = "https://github.com/albertlauncher/python"
+md_maintainers = ["@Bierchermuesli"]
+md_authors = ["@janeklb"]
+md_lib_dependencies = ["nmcli"]
```

Comment:
is this really this one https://pypi.org/project/nmcli/ ? important otherwise dep mgmt will not work

Reply:
wait it does not even use this package. use `md_bin_dependencies`